modules/api/src/main/java/org/eclipse/xpanse/api/ServiceRegisterApi.java

```diff
@@ -54,6 +55,7 @@
 @RestController
 @RequestMapping("/xpanse")
 @CrossOrigin
+@Secured({"admin", "csp"})
```

Comment:
is this annotation ignored by Spring when `zitadel` profile is not enabled?

Reply:
Yes. It only works when the @EnableMethodSecurity(securedEnabled = true) is enabled.